src/Sylius/Bundle/CoreBundle/StateMachine/TransitionInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\StateMachine;
+
+interface TransitionInterface
+{
+    public function getName(): string;
+
+    /**
+     * @return array<string>|null
+     */
+    public function getFroms(): ?array;
+
+    /**
+     * @return array<string>|null
+     */
+    public function getTos(): ?array;
```

Comment:
We want to be compatible with Workflow, so despite Winzou allows to return only 1 "to", we return an array to `to`s.